test/test_postponed_annotations.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
```

Comment:
To be consistent, this could also be moved one level down, next to the `a.py` and `b.py` files.